tests/Keboola/DbExtractorConfig/ConfigTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Tests;
+
+use Keboola\DbExtractorConfig\Config;
+use Keboola\DbExtractorConfig\Configuration\ConfigRowDefinition;
+use Keboola\DbExtractorConfig\Exception\UserException as ConfigUserException;
+use Keboola\DbExtractorConfig\Test\AbstractConfigTest;
+
+class ConfigTest extends AbstractConfigTest
+{
+    public const DRIVER = 'config';
+
+    public function testInvalidConfigTableQuery(): void
+    {
+        $config = $this->getConfig(self::DRIVER);
```

Comment:
IMHO je to špatné použití abstrakce tohle by měl být naprosto čistý test bez jakýchkoli závislostí - ty nikdy nejsou pro parsování konfigurace potřeba, ne? Měl bys tady mít dataprovider na platné a neplatné configy. A ty configy tady mít celé, aby bylo vidět, když se to mění.

Reply:
Vím, že se to jinde takhle dělá, ale tady to mám PRÁVĚ PROTO abychom to oddělili a mohli to hezky v izolaci otestovat.